sourced/ml/utils/engine.py

```diff
@@ -88,52 +41,29 @@ def add_engine_args(my_parser, default_packages=None):
                            help="Print the PySpark execution plans.")
 
 
-def create_spark(session_name,
-                 spark=SparkDefault.MASTER_ADDRESS,
-                 spark_local_dir=SparkDefault.LOCAL_DIR,
-                 config=SparkDefault.CONFIG,
-                 packages=SparkDefault.PACKAGES,
-                 spark_log_level=SparkDefault.LOG_LEVEL,
-                 **_):  # **kwargs are discarded for convenience
-    log = logging.getLogger("spark")
-    log.info("Starting %s on %s", session_name, spark)
-    builder = SparkSession.builder.master(spark).appName(session_name)
-    builder = builder.config(
-        "spark.jars.packages", ",".join(packages))
-    builder = builder.config("spark.local.dir", spark_local_dir)
-    for cfg in config:
-        builder = builder.config(*cfg.split("=", 1))
-    session = builder.getOrCreate()
-    session.sparkContext.setLogLevel(spark_log_level)
-    # Hide py4j verbose logging (It appears in travis mostly)
-    logging.getLogger("py4j").setLevel(logging.WARNING)
-    return session
+def add_engine_dependencies(engine=EngineDefault.VERSION, config=None, packages=None):
+    # TODO(vmarkovtsev): figure out why is this option needed
```

Comment:
I think I asked Alex and DR and they said that it should be enabled for our use cases.